docs/widget-checkbox.md

```diff
@@ -0,0 +1,9 @@
+# [Home](/cogboard/) >> Checkbox widget
+
+#### Description:
+Widget displays one of three status icons:
+* success (green check mark)
+* failure (red cancel icon)
+* indeterminate (indeterminate checkbox icon)
+
+Authenticated user can toggle widget state by clicking its area.
```

Comment:
Anonymous user can do this aswell.

Reply:
Good point, statement fixed - I'll add state autosaving on click (after discussion) in next commit